__tests__/revamp_test.re

```diff
@@ -1,300 +1,445 @@
-open Rebase;
 open Jest;
-open Expect;
 
 let () =
-
-describe("Compiled", () => {
-  test("make - flags", () => {
-    let result = 
-      {js|
+  describe("Compiled", () => {
+    open Expect;
+    open Rebase;
```

Comment:
What's the reason for moving this locally?

Reply:
I was trying isolate the error in the tests below. I though maybe there were some conflicts which there were not. I have removed the local opens. Thanks.